client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -62,6 +67,61 @@ class TitanManageMailboxes extends Component {
 		page( emailManagement( selectedSite.slug, selectedDomainName, currentRoute ) );
 	};
 
+	buildNavItem = ( { context, description, disabled = false, materialIcon, text } ) => ( {
+		description,
+		disabled,
```

Comment:
Could we rename `disabled` to `isDisabled` to be more consistent with how we name boolean variables?

Reply:
Sure